copy/multiple_test.go

```diff
@@ -0,0 +1,36 @@
+package copy
+
+import (
+	"testing"
+
+	digest "github.com/opencontainers/go-digest"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPrepareCopyInstances(t *testing.T) {
+	// Test CopyAllImages
+	options := Options{}
+	instances := []digest.Digest{
```

Comment:
(Absolutely non-blocking: Maybe
```suggestion
	sourceInstances := []digest.Digest{
```
)